inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/FieldFormatUtils.java

```diff
@@ -78,4 +89,59 @@ public static String createStructFormat(List<Element> elements) {
     public static StructFormat parseStructFormat(String formatJson) {
         return JsonUtils.parseObject(formatJson, StructFormat.class);
     }
+
+    /**
+     * Parse format
+     *
+     * @param serializationType data serialization, support: csv, json, canal, avro, etc
+     * @param wrapWithInlongMsg whether wrap content with {@link InLongMsgFormat}
+     * @param separatorStr the separator of data content
+     * @param ignoreParseErrors whether ignore deserialization error data
+     * @return the format for serialized content
+     */
+    public static Format parsingFormat(
```

Comment:
I think this function has nothing to do with `FieldFormatUtils` and should be placed in `ExtractNodeProvider`

Reply:
> I think this function has nothing to do with `FieldFormatUtils` and should be placed in `ExtractNodeProvider`

FieldFormatUtils also contains Format, but it is not very accurate, it can be placed in ExtractNodeProvider